package.json

```diff
@@ -37,15 +38,16 @@
     "version": "3.2.2"
   },
   "scripts": {
-    "bumpdocs": "npm cache clean aframevr/aframe#docs-v0.4.0 && rm -rf .multidep/aframe-v0.4.0 && npm install aframevr/aframe#master && multidep multidep.json",
+    "bumpdocs": "npm cache clean aframevr/aframe#docs-v0.4.0 && rm -rf .multidep/aframe-v0.4.0 && npm install aframevr/aframe#master && npm run copy-docs && smultidep multidep.json",
```

Comment:
typo at the end: `smultidep`. The copy docs should be run after the install. also it got hyphenated which isn't reflected in the `dev` script (which should be removed) and inconsistent with JS.